### ISSUE TITLE
Bust the Maven cache when checking for new releases

### ISSUE THREAD

**Issue by st3fan** [mozilla-mobile/relbot#38]
Just noticed a case where ...

> `https://maven.mozilla.org/maven2/org/mozilla/components/ui-widgets/maven-metadata.xml`

returned a list of versions that was missing `67.0.12`, while the following returned the correct metadata ...

> `https://maven.mozilla.org/maven2/org/mozilla/components/ui-widgets/maven-metadata.xml?t=282798273`

Maybe but the cache with a `t=timestamp` query parameter?

**Comment by st3fan** [comment]
@pocmo this may explain some random build errors that we've been seeing where artifacts cannot be found.

**Comment by st3fan** [comment]
Before:

```
2020-12-15 03:53:56.744758 Looking at Fenix 84
2020-12-15 03:53:56.830408 Looking at Fenix 84 on releases/v84.0.0
2020-12-15 03:53:56.923890 Current A-C version in Fenix is 67.0.11
2020-12-15 03:53:56.997144 Latest A-C version available is 67.0.11
2020-12-15 03:53:56.997160 No need to upgrade; Fenix 84 is on A-C 67.0.11
```

Womp womp womp .. it is there but Maven doesn't tell us ... 

After:

```
2020-12-15 04:36:07.062277 Looking at Fenix 84 on releases/v84.0.0
2020-12-15 04:36:07.186082 Current A-C version in Fenix is 67.0.11
2020-12-15 04:36:07.456222 Latest A-C version available is 67.0.12
2020-12-15 04:36:07.456251 We are going to upgrade Fenix 84 to Android-Components 67.0.12
2020-12-15 04:36:07.549496 Last commit on releases/v84.0.0 is cdc02def7468c82a6da9365082040bc218490fa5
2020-12-15 04:36:07.549524 Creating branch relbot/update-android-components/Fenix-84/AC-67.0.12 on cdc02def7468c82a6da9365082040bc218490fa5
2020-12-15 04:36:08.186064 Updating AndroidComponents.kt from 67.0.11 to 67.0.12 on relbot/update-android-components/Fenix-84/AC-67.0.12
2020-12-15 04:36:09.059597 Creating pull request
2020-12-15 04:36:10.045508 Pull request at https://github.com/mozilla-mobile/fenix/pull/17040
```
